src/constants.ts

```diff
@@ -38,3 +38,9 @@ export const SNIPPETS_URL = "https://raw.githubusercontent.com/spicetify/spiceti
 export const SNIPPETS_PAGE_URL = "https://github.com/spicetify/spicetify-marketplace/blob/main/resources/snippets.json";
 
 export const BLACKLIST_URL = "https://raw.githubusercontent.com/spicetify/spicetify-marketplace/main/resources/blacklist.json";
+
+export const REMOTE_VERSION = "https://raw.githubusercontent.com/spicetify/spicetify-marketplace/main/package.json";
+
+// Used for Changelog
+const commitTimestamp = "2022-06-07T21:05:22Z"; // Must be in ISO format
```

Comment:
Will this need to be updated whenever we bump a version as well? I was thinking we do the commits instead of manually specifying changelog notes to save us from having the change things in a bunch of places. Is it possible to just grab the latest release from GitHub and use the notes from that? I'm also realizing we probably don't want to notify people every time we bump the version without a release, since I've been trying to version bump more often when we add a new feature or something. Or we do releases more frequently? Hope this makes some sense, just kind of thinking out loud.

Reply:
It does, yeah.
I also thought about fetching changelogs and release notes from GitHub but we barely push or publish there (already skipped 0.6.0 and 0.6.1) so it didn't seem plausible for me.

Reply:
If you do decide to go along with publishing releases more frequently then I have already implemented the feature just in case.
![image](https://user-images.githubusercontent.com/77577746/173241397-b0100087-f3ab-4075-a151-4795313d225c.png)